frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -1,27 +1,24 @@
 import qs from "querystring";
 
 import { MockedProvider } from "@apollo/client/testing";
-import {
-  render,
-  screen,
-  waitFor,
-  waitForElementToBeRemoved,
-  within,
-} from "@testing-library/react";
+import { render, screen, waitFor, within } from "@testing-library/react";
```

Comment:
apologies for the whitespace diffs here! ran the commit hooks a couple times and still couldn't get the linters to work like you would expect them to.